types/recharts/index.d.ts

```diff
@@ -151,11 +152,11 @@ export interface CartesianAxisProps {
 }
 export class CartesianAxis extends React.Component<CartesianAxisProps, {}> {}
 
-export interface CartesianGridProps {
+export interface CartesianGridProps extends Partial<CSSStyleDeclaration> {
 	x?: number;
 	y?: number;
-	width?: number;
-	height?: number;
+	// width?: number;
+	// height?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?